mmv1/templates/terraform/constants/notebooks_instance.go

```diff
@@ -33,3 +33,56 @@ func NotebooksInstanceKmsDiffSuppress(_, old, new string, _ *schema.ResourceData
 	}
 	return false
 }
+
+<% unless compiler == "terraformgoogleconversion-codegen" -%>
```

Comment:
Any particular reason to wrap this function with <% unless compiler == "terraformgoogleconversion-codegen" -%>?

Reply:
I am actually not familiar with the reason this is required. If this is not present, I had the conversion test fail for workbench PR, so I added it here as well.